pkg/appmgmt/general/general.go

```diff
@@ -130,6 +130,7 @@ func (os *Manager) getAppMetadata(pod *v1.Pod) (interfaces.ApplicationMetadata,
 		User:          user,
 		Tags:          tags,
 		TaskGroups:    taskGroups,
+		PlaceholderOwnerReferences: pod.OwnerReferences,
```

Comment:
I think we can just call this owner reference since this is a general K8s concept.
It doesn't mean only placeholder pod has that.